conf/supervisor/defineimage.conf

```diff
@@ -0,0 +1,22 @@
+[program:defineimage]
```

Comment:
This and the other two participants should not go in. defineimage and do_build_trial got removed in git history. wait_for_repo_published is unused (perhaps it shares ancestry with is_repo_published?)